omc3/plotting/plot_checked_corrections.py

```diff
@@ -403,6 +429,72 @@ def save_plots(output_dir, figure_dict, input_dir=None):
             fig.savefig(output_path)
 
 
+def show_plots(figure_dict: Dict[str, Figure]):
+    """ Shows plots.
+    If qtpy is installed, they are shown in a single window.
+    The individual corrections are sorted in to vertical tabs,
+    the optics parameter into horizontal tabs. 
+    If qtpy is not installed, they are simply shown as individual figures.
+    This is not recommended
+    """
+    try:
+        window = VerticalTabWindow("Correction Check")
+    except TypeError:
+        log_no_qtpy_many_windows()
+        for name, fig in figure_dict.items():
+            create_pyplot_window_from_fig(fig, title=name.replace(SPLIT_ID, " "))
+        plt.show()
+        return
+
+    rdt_pattern = re.compile(r"f\d{4}")
+    rdt_complement = {
+        RDT_REAL_COLUMN.text_label: RDT_IMAG_COLUMN,
+        RDT_AMPLITUDE_COLUMN.text_label: RDT_PHASE_COLUMN,
+    }
+
+    correction_names = sorted(set([n.split(SPLIT_ID)[0] for n in figure_dict.keys() if SPLIT_ID in n]))
+    for correction_name in [None] + list(correction_names):
+        if not correction_name:
+            parameter_names = iter(sorted(n for n in figure_dict.keys() if SPLIT_ID not in n))
+            correction_tab_name = "All Corrections"
+        else:
+            parameter_names = iter(sorted(n for n in figure_dict.keys() if correction_name in n))
+            correction_tab_name = correction_name
+
+        current_tab = TabWidget(title=correction_tab_name)
+        window.add_tab(current_tab)
+
+        for name_x in parameter_names:
+            tab_prename = name_x.split(SPLIT_ID)[-1]
```

Comment:
maybe add a comment to explain which part you're getting, not clear to me by just reading the code

Reply:
Well, that depends... because of the way plot-tfs creates unique IDs and what we are plotting.
There is two kinds of plots:
a) Plots showing the expected value after correction + the measured value. In this cases the unique id is created by the file-name the value comes from.
b) Plots showing expectation, measurement and correction for a single Correction. In this case all of these come from the same file, so the unique name is filename_columnname

Reply:
I actually explained it the wrong way around... but yeah... it's the ID and I am trying to extract a useful tab-name from there